README.md

```diff
@@ -25,21 +25,29 @@ Usage
 Install
 =======
 
- Officially only linux operating systems are supported, although users have had success installing on windows too.
+Officially only the Linux operating systems are supported, although users have had success installing on Windows too.
  
- The default installation will place headers in /usr/local/include and
-shared library in /usr/local/lib. It also installs a pkg-config script
-into /usr/local/lib/pkgconfig and will install a python wrapper if python3 is installed.
-
-If you have CMake installed or it is not difficult to install, then do:
-
-    $ cmake .
-    $ sudo make install
-    
+The default installation will place headers in /usr/local/include and shared library in /usr/local/lib. It also installs a pkg-config script into /usr/local/lib/pkgconfig and will install a python wrapper if python3 is installed.
+
+## cmake
```

Comment:
Not everybody has ninja installed so it still makes sense for people to use `mkdir build ; cd build ; cmake .. -DCMAKE_BUILD_TYPE=Release ; make -j && make install`. We should recommend the use of ninja though, and then `cmake .. -GNinja -DCMAKE_BUILD_TYPE=Release ; ninja ; ninja install` should be enough (even though not pure CMake)

Reply:
Alright. I change the documentation back to using the default Makefile generator. The compilation via  `cmake --build build --target install` is still the same (it's the same for any generator).